### PR TITLE
Release/1.6.0

### PR DESCRIPTION
**Pull request by QuintinWillison**

### CODE REVIEW

**Comment by QuintinWillison** [comment]
Sorry about https://github.com/ably/ably-asset-tracking-android/pull/971/commits/9e23f61d1dc6ec39cd49b170a691f88c0ef39013, I either failed to pull in latest changes before I branched, or it moved while I was working on this PR.

**Review by AndyTWF — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,10 +16,12 @@ The current, non-pre-release SDKs for production use are at version [1.5.1](#151
...
 **Fixed bugs:**
 
+- Exceptions thrown for new publishing requests when offline [\#871](https://github.com/ably/ably-asset-tracking-android/issues/871)
 - Retry behaviour improvements for Ably API calls [\#927](https://github.com/ably/ably-asset-tracking-android/issues/927)
-- `Publisher.remove()` fails during several connectivity faults [\#905](https://github.com/ably/ably-asset-tracking-android/issues/905)
+- `Publisher.remove\(\)` fails during several connectivity faults [\#905](https://github.com/ably/ably-asset-tracking-android/issues/905)
 - Adding a trackable stalls forever if presence.enter\(\) is interrupted by a disconnection [\#896](https://github.com/ably/ably-asset-tracking-android/issues/896)
 - Publisher crashes when location data has NaN value [\#861](https://github.com/ably/ably-asset-tracking-android/issues/861)
+- Subscriber, upon losing connectivity, continues to show the Publisher as online [\#835](https://github.com/ably/ably-asset-tracking-android/issues/835)
 - Subscriber, upon losing connectivity, continues to show the Publisher as online [\#833](https://github.com/ably/ably-asset-tracking-android/issues/833)
```
Comment (KacperKluka) [inline CHANGELOG.md:25]: The duplicate entry strikes again 😅

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 
-The current, non-pre-release SDKs for production use are at version [1.5.1](#151).
+## [1.6.0](https://github.com/ably/ably-asset-tracking-android/tree/v1.6.0)
 
-## [1.6.0, Beta 1](https://github.com/ably/ably-asset-tracking-android/tree/v1.6.0-beta.1)
+This release enhances operational stability, for publishers in particular, when running on a devices with unreliable network connectivity.
 
-[Full Changelog](https://github.com/ably/ably-asset-tracking-android/compare/v1.5.1...v1.6.0-beta.1)
+Please see:
+[Upgrade / Migration Guide from v.1.5.1](https://github.com/ably/ably-asset-tracking-android/blob/main/UPDATING.md#version-151-to-160)
```
Comment (KacperKluka) [inline CHANGELOG.md:8]: The migration guide is missing a few important things for users, for example, no longer throwing exceptions while `add/track` and instead operating on the `TrackableState` flow. I know that we're in a rush so perhaps at least let's create an issue to add the missing details later? 🤔

**Review by KacperKluka — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -13,10 +16,12 @@ The current, non-pre-release SDKs for production use are at version [1.5.1](#151
 
 **Fixed bugs:**
 
+- Exceptions thrown for new publishing requests when offline [\#871](https://github.com/ably/ably-asset-tracking-android/issues/871)
 - Retry behaviour improvements for Ably API calls [\#927](https://github.com/ably/ably-asset-tracking-android/issues/927)
-- `Publisher.remove()` fails during several connectivity faults [\#905](https://github.com/ably/ably-asset-tracking-android/issues/905)
+- `Publisher.remove\(\)` fails during several connectivity faults [\#905](https://github.com/ably/ably-asset-tracking-android/issues/905)
```
Comment (KacperKluka) [inline CHANGELOG.md:21]: The backslashes also strike again 😉

**Review by KacperKluka — APPROVED** [review]
Due to the fact that we're in a bit of a hurry, I will approve the PR despite the small things I've pointed out before 😉 @QuintinWillison feel free to resolve my previous comments 👍

File: CHANGELOG.md (hunk 2 above)
Comment (QuintinWillison) [reply CHANGELOG.md:8]: Deferred: https://github.com/ably/ably-asset-tracking-android/issues/974

File: CHANGELOG.md (hunk 3 above)
Comment (QuintinWillison) [reply CHANGELOG.md:21]: Deferred: https://github.com/ably/ably-asset-tracking-android/issues/975

File: CHANGELOG.md (hunk 1 above)
Comment (QuintinWillison) [reply CHANGELOG.md:25]: Deferred into scope of: https://github.com/ably/ably-asset-tracking-android/issues/975#issue-1564074594

**Pull request merged**